@navikt/core/react/src/form/form-summary/FormSummaryFooter.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef } from "react";
+
+export interface FormSummaryFooterProps
+  extends React.HTMLAttributes<HTMLElement> {
+  children: React.ReactNode;
+}
+
+export const FormSummaryFooter = forwardRef<
+  HTMLElement,
+  FormSummaryFooterProps
+>(({ children, className, ...rest }, ref) => (
+  <footer
+    ref={ref}
+    {...rest}
+    className={`navds-form-summary__footer${className ? ` ${className}` : ""}`}
```

Comment:
We normally use the package `clsx` to handle className collections. Since we currently support two versions of our CSS-package, we have wrapped this within the hook 
```
const { cn } = useRenameCSS();
```

```
    className={cn("navds-form-summary__footer", className)}

```